portfolio/src/main/java/com/google/sps/data/Video.java

```diff
@@ -7,16 +7,29 @@
 
 // Represents a video, including its ID, title, description, and captions.
 public class Video {
+    // Fetch via the youtube API before previewing.
     private String videoId;
     private String title;
     private String description;
+    private String thumbnailUrl;
```

Comment:
Would it be better to use this class? https://docs.oracle.com/javase/7/docs/api/java/net/URL.html

Reply:
Updated to use the URL class. While testing I noticed that malformed inputs redirect to a "not found" page since forms always auto redirect. This can be fixed by making a separate request outside of the form (I did this in my starter project) but I'm wondering whether I should here or in a separate PR.

Reply:
WIll handle in separate PR.